### PR TITLE
[file_length_] A versatile plugin to count the lines in specified files

### PR DESCRIPTION
**Pull request by shtrom**
Signed-off-by: Olivier Mehani <shtrom@ssji.net>

### CODE REVIEW

File: plugins/system/file_length_ [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```
Comment (sumpfralle) [inline plugins/system/file_length_]: /bin/h should be sufficient.

File: plugins/system/file_length_ [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+=cut
+
+#NAME=`echo $0 | sed 's/.*_//'`
+NAME=${0#*_}
+TITLE=${title:-File lengths for $NAME}
+CATEGORY=${category:-system}
+
+FILES=${files:-/var/log/messages}
+FILES=$(echo $(ls $FILES))
```
Comment (sumpfralle) [inline plugins/system/file_length_]: `echo $FILES` should be sufficient for globbing (if this your point). Please add a comment regarding the purpose of this line.

File: plugins/system/file_length_ [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	cat <<EOF
+graph_title ${TITLE}
+graph_args ${graph_args} --base 1000
+graph_category ${CATEGORY}
+graph_info This graph shows the length of ${FILES}
+graph_vlabel length (lines)
+EOF
+
+	for F in $FILES; do
+		MF=`echo $F | sed 's/[-\/\.]/_/g'`
```
Comment (sumpfralle) [inline plugins/system/file_length_]: Please use "clean_fieldname" from "$MUNIN_LIBDIR/plugins/plugin.sh" instead.

File: plugins/system/file_length_ (hunk 1 above)
Comment (shtrom) [inline plugins/system/file_length_]: Nope, needed for `echo -n`, otherwise `shellcheck` complains (:

Comment (shtrom) [inline plugins/system/file_length_]: Ah, actually, no, I know how to work around that.

**Comment by shtrom** [comment]
Should be good for another look.

**Pull request merged**